Foundation/NSOrderedSet.swift

```diff
@@ -252,22 +228,16 @@ extension NSOrderedSet {
     }
     
     public func objectEnumerator() -> NSEnumerator {
-        guard type(of: self) === NSOrderedSet.self || type(of: self) === NSMutableOrderedSet.self else {
-            NSRequiresConcreteImplementation()
-        }
-        return NSGeneratorEnumerator(_orderedStorage.map { __SwiftValue.fetch(nonOptional: $0) }.makeIterator())
+        return _orderedStorage.objectEnumerator()
```

Comment:
This has the interesting side effect of making these enumerators behave similar to Darwin - they will now break if mutation occurs during iteration. The previous implementation made a copy of the all the objects using `map` and therefore did not have this issue.